framework/base/Model.php

```diff
@@ -239,9 +239,15 @@ public function scenarios()
      */
     public function formName()
     {
-        $reflector = new ReflectionClass($this);
+        static $formName = null;
```

Comment:
@laszlovl Thanks, that's quite interesting. I've never really thought about what happens to `static` variables in class methods. I wonder if this behavior is documented in the PHP manual somewhere. I would have expected it [here](http://php.net/manual/en/language.variables.scope.php#language.variables.scope.static), but it's missing.